app/domain/appeal-application.ts

```diff
@@ -0,0 +1,44 @@
+interface AppealApplication {
```

Comment:
Can we move this to types folder to have all the interfaces and types in one place?